add_to_app/multiple_flutters/README.md

```diff
@@ -0,0 +1,80 @@
+# multiple_flutters
+
+This is a sample that shows how to use the Flutter Engine Groups API to embed
+multiple instances of Flutter into an existing Android or iOS project.
+
+## Getting Started
+
+For iOS instructions, see:
+[multiple_flutters_ios](../multiple_flutters_ios/README.md).
```

Comment:
nit: Are the `../` paths still necessary with the new folder structure?

Reply:
done